deploy/ansible/roles/iroha-cluster-deploy-node/templates/config.j2

```diff
@@ -0,0 +1,10 @@
+{
+  "block_store_path" : "/tmp/block_store/",
+  "torii_port" : {{ torii_port + item | int }},
+  "internal_port" : {{ internal_port + item | int }},
+  "pg_opt" : "host={{ postgresName }}_{{ key + item | int }}_1 port=5432 user={{ postgresUser }} password='{{ postgresPassword }}'",
+  "max_proposal_size" : 10,
```

Comment:
In the future, we need all the numeric values to be configurable individually per each peer. This is not needed now, but can you please create a task? :)